controllers/glance_common.go

```diff
@@ -19,10 +19,11 @@ package controllers
 import (
 	"context"
 	"fmt"
+	"time"
```

Comment:
not required as long as you use `ResultRequeue`: https://github.com/openstack-k8s-operators/glance-operator/blob/main/pkg/glance/const.go#L121